facia/app/controllers/FaciaController.scala

```diff
@@ -207,7 +207,7 @@ trait FaciaController
       case Some((faciaPage, _)) if nonHtmlEmail(request) =>
         successful(Cached(CacheTime.RecentlyUpdated)(renderEmail(faciaPage)))
       case Some((faciaPage: PressedPage, targetedTerritories))
-          if FaciaPicker.getTier(faciaPage, path)(request) == RemoteRender
+          if FaciaPicker.getTier(faciaPage)(request) == RemoteRender
```

Comment:
**non-blocking** `request` is implicit here and in `getTier` so it doesn't need to be passed